guides/source/configuring.md

```diff
@@ -1124,6 +1124,29 @@ These configuration points are then available through the configuration object:
   Rails.configuration.x.super_debugger.not_set      # => nil
   ```
 
+You can also use `Rails::Application.config_for` to load whole configuration files:
+
+  ```ruby
+  # config/my_app.yml:
+  production:
+    url: http://127.0.0.1:8080
+    namespace: my_app_production
+  development:
+    url: http://localhost:3001
+    namespace: my_app_development
+
+  # config/application.rb
+  module MyApp
+    class Application < Rails::Application
+      config.x = Rails.application.config_for(:my_app)
```

Comment:
The `x` namespacing is no longer required in Rails 5 - can you change to something specific, e.g:

``` ruby
config.payment = Rails.application.config_for(:payment)
```